example/alertmanager-config.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: alertmanager-main
+data:
+  alertmanager.yaml: |-
+    global:
+      resolve_timeout: 5m
+    route:
+      group_by: ['alertname']
```

Comment:
I've done this wrong myself in the past, but I think the default grouping should most likely go by `job`.